tests/MongoDB.Driver.Examples/CausalConsistencyExamples.cs

```diff
@@ -0,0 +1,130 @@
+                var items = client.GetDatabase("test", new MongoDatabaseSettings
+                    {
+                        ReadConcern = ReadConcern.Majority,
+                        WriteConcern = new WriteConcern(
+                            WriteConcern.WMode.Majority,
+                            TimeSpan.FromMilliseconds(1000))
+                    })
+                    .GetCollection<BsonDocument>("items");
+
+                items.UpdateOne(session1,
+                    Builders<BsonDocument>.Filter.And(new BsonDocument("sku", 111), new BsonDocument("end", BsonNull.Value)),
+                    Builders<BsonDocument>.Update.Set("end", currenDate));
+
+                items.InsertOne(session1, new BsonDocument
+                {
+                    {"sku", "nuts-111"},
+                    {"name", "Pecans"},
+                    {"start", currenDate}
+                });
+            }
+            //End Causal Consistency Example 1
+
+            var result = testDatabase.GetCollection<BsonDocument>(itemsCollectionName).Find("{}").FirstOrDefault();
+            RemoveIds(new[] { result });
+            result["start"].Should().NotBeNull();
+            result.Remove("start");
+            result.Should().Be("{ \"sku\" : \"nuts-111\", \"name\" : \"Pecans\" }");           
+        }
+
+        [Fact]
+        public void Causal_Consistency_Example_2()
+        {
+            RequireServer.Check().SupportsCausalConsistency();
+
+            var client = GetClient();
+
+            string itemsCollectionName = "items";
+            var testDatabase = client.GetDatabase("test");
+            testDatabase.DropCollection(itemsCollectionName);
+
+            using (var session1 = client.StartSession(new ClientSessionOptions { CausalConsistency = true }))
+            {
+                var currenDate = DateTime.UtcNow.Date;
+                var items = client.GetDatabase("test", new MongoDatabaseSettings
+                    {
+                        ReadConcern = ReadConcern.Majority,
+                        WriteConcern = new WriteConcern(
+                            WriteConcern.WMode.Majority,
+                            TimeSpan.FromMilliseconds(1000))
+                    })
+                    .GetCollection<BsonDocument>("items");
+
+                items.InsertOne(session1, new BsonDocument
+                {
+                    {"sku", "nuts-111"},
+                    {"name", "Pecans"},
+                    {"start", currenDate}
+                });
+
+                // Start Causal Consistency Example 2
+                using (var session2 = client.StartSession(new ClientSessionOptions { CausalConsistency = true }))
+                {
+                    session2.AdvanceClusterTime(session1.ClusterTime);
+                    session2.AdvanceOperationTime(session1.OperationTime);
+
+                    var items2 = client.GetDatabase("test", new MongoDatabaseSettings
```

Comment:
This variable should be called "items" to match the requested example.

Rename the other variable to avoid the name clash.